Cargo.lock

```diff
@@ -3590,13 +3590,13 @@ checksum = "3576a87f2ba00f6f106fdfcd16db1d698d648a26ad8e0573cad8537c3c362d2a"
 
 [[package]]
 name = "lexical-core"
-version = "0.7.5"
+version = "0.7.4"
```

Comment:
run `cargo update -p lexical-core:0.7.4` and this should fix the nightly build check.

Reply:
I cannot update to 0.7.4, it always updates to 0.7.6, also crates.io page lists 0.7.4 in gray: https://crates.io/crates/lexical-core/versions Can we go with 0.7.6?

Reply:
Well the issue is that you're on 0.7.4 right now. `cargo update -p lexical-core:0.7.4` upgrades it to the newest version according to semvar. So yeah, the point of that command IS to update to 0.7.6.

Reply:
Ah I see. It's fixed.